extensions/amp-base-carousel/1.0/arrow.js

```diff
@@ -85,6 +85,7 @@ function DefaultArrow({by, ...rest}) {
   const classes = useStyles();
   return (
     <button
+      part={`arrow ${by > 0 ? 'next' : 'prev'}`}
```

Comment:
Do we want to use spaces in the part name?

Reply:
`part` really functions like a `class` attribute, except that the names are more logical. So, this declares two parts on the element: "arrow" and "next" (or "prev"). In CSS they can be used independently or together:

```
.carousel::part(arrow) {
  /* styles for both arrows: next and prev */
}
.carousel::part(arrow next) {
  /* styles for next arrow only */
}
```